## 78% of end-end tests have passed
```
Total executed tests: 23
Failed tests: 5
Expected to fail tests: 0
Passed tests: 18
Not executed tests: 1
```
```
IDs of not executed tests: 742910 
```
```
IDs of failed tests: 740490,741612,703391,702731,702730 
```
<h3>Not executed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/priority-fee-current&PR_ID=22568&APK_NAME=22568.apk&TR_CASE_IDS=742910">Rerun not executed tests</a></li></details><h3>Failed tests (5)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/priority-fee-current&PR_ID=22568&APK_NAME=22568.apk&TR_CASE_IDS=740490,741612,703391,702731,702730">Rerun failed tests</a></li><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855027">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `network-dropdown` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
wallet/test_wallet_mainnet_no_send.py:70: in test_wallet_balance_mainnet
    self.errors.verify_no_errors()
base_test_case.py:200: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: For the Ether the wrong value 0.0362 is shown, expected 0.015 on Base
E    Device 1: For the Status the wrong value 15.0 is shown, expected 0.0 on Base
E    Device 1: For the Uniswap the wrong value 0.7 is shown, expected 0.0 on Base
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESq6r88Y6aslf9n5rA">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855030">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `SlideButton` by `xpath`: `//*[@resource-id='slide-button-track']` </div><div>Device 1: Click system back button </div></blockquote></p>

```
wallet/test_wallet_mainnet_no_send.py:349: in test_wallet_bridge_flow_mainnet
    self.errors.verify_no_errors()
base_test_case.py:200: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Base to Optimism: Max fees is not shown before pressing Review Bridge button
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESq6r88Y6aslf9n5rA">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855023">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
Test setup failed: private_chats/test_one_to_one_chat.py:39: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:387: in handle_contact_request
    self.open_activity_center_button.click_until_presence_of_element(self.close_activity_centre)
../views/base_element.py:107: in click_until_presence_of_element
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `open-activity-center-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855016">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test setup failed: private_chats/test_one_to_one_chat.py:39: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:387: in handle_contact_request
    self.open_activity_center_button.click_until_presence_of_element(self.close_activity_centre)
../views/base_element.py:107: in click_until_presence_of_element
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `open-activity-center-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855015">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `Button` by `accessibility id`: `close-activity-center` will be presented </div><div>Device 2: Find `Button` by `accessibility id`: `open-activity-center-button` </div></blockquote></p>

```
Test setup failed: private_chats/test_one_to_one_chat.py:39: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:387: in handle_contact_request
    self.open_activity_center_button.click_until_presence_of_element(self.close_activity_centre)
../views/base_element.py:107: in click_until_presence_of_element
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `open-activity-center-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Might fail from time to time as reaction are set too slowly]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES7nXHeafukqnwa0oE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmEDUVZdkbfsQskiW">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (18)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855032">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESneiMn0ZytpoGuBym">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855031">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESneiMn0ZytpoGuBym">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855033">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESneiMn0ZytpoGuBym">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855029">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESq6r88Y6aslf9n5rA">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855028">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESq6r88Y6aslf9n5rA">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855020">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyhFgeDES13d5baQs">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStrUO3d1bxdzNsqVW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855019">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyhFgeDES13d5baQs">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStrUO3d1bxdzNsqVW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855022">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyhFgeDES13d5baQs">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStrUO3d1bxdzNsqVW">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855021">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyhFgeDES13d5baQs">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEStrUO3d1bxdzNsqVW">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855018">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESDkHuttfgFiNk4Z5F">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaTcE1txbtQt9PYMu">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESkrmNMLGEcZfm3SGI">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855026">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESvniIyk1Y0WPhfQUJ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855017">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZWDogCylsURM79PE">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855035">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESFYQGloPd4kMWPpSM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPJJLRgj78atfMTn5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855034">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESFYQGloPd4kMWPpSM">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPJJLRgj78atfMTn5">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855024">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES61XkpYmcuFPkPaeE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESSwR3v9Ek0zfeqPnb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855025">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES61XkpYmcuFPkPaeE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESSwR3v9Ek0zfeqPnb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855036">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES61XkpYmcuFPkPaeE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESSwR3v9Ek0zfeqPnb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3855037">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES61XkpYmcuFPkPaeE">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESSwR3v9Ek0zfeqPnb">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/common/transaction_settings/max_fee/view.cljs
+++ b/src/status_im/contexts/wallet/common/transaction_settings/max_fee/view.cljs
@@ -29,11 +29,11 @@
   []
   (let [network-base-fee (rf/sub [:wallet/tx-settings-network-base-fee-route])
         max-base-fee     (rf/sub [:wallet/tx-settings-max-base-fee])
-        priority-fee     (rf/sub [:wallet/tx-settings-priority-fee])
+        priority-fee     (rf/sub [:wallet/tx-settings-custom-priority-fee])
         conditions       (partial hint-and-status network-base-fee priority-fee)]
     [transaction-settings/custom-setting-screen
      {:screen-title  (i18n/label :t/max-base-fee)
-      :token-sybmol  :gwei
+      :token-symbol  :gwei
       :conditions-fn conditions
       :current       max-base-fee
       :info-title    (i18n/label :t/max-base-fee)

--- a/src/status_im/contexts/wallet/common/transaction_settings/priority_fee/view.cljs
+++ b/src/status_im/contexts/wallet/common/transaction_settings/priority_fee/view.cljs
@@ -6,7 +6,7 @@
     [utils.re-frame :as rf]))
 
 (defn hint-and-status
-  [priority-fee spectrum max-base-fee entered-value]
+  [spectrum max-base-fee entered-value]
   (let [upper-limit-exceeded? (> entered-value (:high spectrum))
         lower-limit-exceeded? (< entered-value (:low spectrum))]
     (cond
@@ -19,8 +19,7 @@
                                       :status    :warning}
       lower-limit-exceeded?          {:hint-text (i18n/label :t/priority-fee-lower spectrum)
                                       :status    :warning}
-      :else                          {:hint-text (i18n/label :t/priority-fee-current
-                                                             {:current priority-fee})
+      :else                          {:hint-text (i18n/label :t/priority-fee-current spectrum)
                                       :status    :default})))
 
 (defn view
@@ -29,7 +28,7 @@
         max-base-fee (rf/sub [:wallet/tx-settings-max-base-fee])
         spectrum     {:low  (rf/sub [:wallet/tx-settings-suggested-min-priority-fee])
                       :high (rf/sub [:wallet/tx-settings-suggested-max-priority-fee])}
-        conditions   (partial hint-and-status priority-fee spectrum max-base-fee)]
+        conditions   (partial hint-and-status spectrum max-base-fee)]
     [transaction-settings/custom-setting-screen
      {:screen-title  (i18n/label :t/priority-fee)
       :token-sybmol  :gwei

--- a/src/status_im/contexts/wallet/common/transaction_settings/priority_fee/view.cljs
+++ b/src/status_im/contexts/wallet/common/transaction_settings/priority_fee/view.cljs
@@ -24,14 +24,14 @@
 
 (defn view
   []
-  (let [priority-fee (rf/sub [:wallet/tx-settings-priority-fee])
+  (let [priority-fee (rf/sub [:wallet/tx-settings-custom-priority-fee])
         max-base-fee (rf/sub [:wallet/tx-settings-max-base-fee])
         spectrum     {:low  (rf/sub [:wallet/tx-settings-suggested-min-priority-fee])
                       :high (rf/sub [:wallet/tx-settings-suggested-max-priority-fee])}
         conditions   (partial hint-and-status spectrum max-base-fee)]
     [transaction-settings/custom-setting-screen
      {:screen-title  (i18n/label :t/priority-fee)
-      :token-sybmol  :gwei
+      :token-symbol  :gwei
       :conditions-fn conditions
       :current       priority-fee
       :info-title    (i18n/label :t/priority-fee)

--- a/src/status_im/contexts/wallet/common/transaction_settings/view.cljs
+++ b/src/status_im/contexts/wallet/common/transaction_settings/view.cljs
@@ -12,7 +12,7 @@
 (defn custom-settings-sheet
   [_]
   (let [max-base-fee   (rf/sub [:wallet/tx-settings-max-base-fee])
-        priority-fee   (rf/sub [:wallet/tx-settings-priority-fee])
+        priority-fee   (rf/sub [:wallet/tx-settings-custom-priority-fee])
         max-gas-amount (rf/sub [:wallet/tx-settings-gas-amount])
         nonce          (rf/sub [:wallet/tx-settings-nonce])
         account-color  (rf/sub [:wallet/current-viewing-account-color])

--- a/src/status_im/subs/wallet/wallet.cljs
+++ b/src/status_im/subs/wallet/wallet.cljs
@@ -978,6 +978,18 @@
    (or value-set-by-user (:tx-priority-fee gas-fees))))
 
 (rf/reg-sub
+ :wallet/tx-settings-custom-priority-fee
+ :<- [:wallet/tx-settings-fee-mode]
+ :<- [:wallet/tx-settings-gas-fees]
+ :<- [:wallet/tx-settings-priority-fee-user]
+ :<- [:wallet/tx-settings-suggested-min-priority-fee]
+ (fn [[fee-mode gas-fees value-set-by-user min-priority-fee]]
+   (cond
+     value-set-by-user                value-set-by-user
+     (= :tx-fee-mode/custom fee-mode) (:tx-priority-fee gas-fees)
+     :else                            min-priority-fee)))
+
+(rf/reg-sub
  :wallet/tx-settings-gas-amount
  :<- [:wallet/tx-settings-gas-amount-route]
  :<- [:wallet/tx-settings-gas-amount-user]

--- a/src/status_im/subs/wallet/wallet.cljs
+++ b/src/status_im/subs/wallet/wallet.cljs
@@ -1012,9 +1012,8 @@
 (rf/reg-sub
  :wallet/tx-settings-suggested-max-priority-fee
  :<- [:wallet/tx-settings-gas-fees]
- :<- [:wallet/tx-settings-max-base-fee]
- (fn [[gas-fees max-base-fee]]
-   (min max-base-fee (:suggested-max-priority-fee gas-fees))))
+ (fn [gas-fees]
+   (:suggested-max-priority-fee gas-fees)))
 
 (rf/reg-sub
  :wallet/tx-settings-suggested-min-priority-fee

--- a/translations/en.json
+++ b/translations/en.json
@@ -2096,7 +2096,7 @@
   "principles": "Principles",
   "priority": "Priority",
   "priority-fee": "Priority fee",
-  "priority-fee-current": "Current: {{current}}",
+  "priority-fee-current": "Current: {{low}} - {{high}}",
   "priority-fee-higher": "Higher than necessary: {{low}} - {{high}}",
   "priority-fee-higher-max-base": "Higher than max base fee: {{max-base-fee}}",
   "priority-fee-lower": "Lower than suggested: {{low}} - {{high}}",

